README.md

```diff
@@ -1,7 +1,13 @@
 # about-me
+answer-one
+I grew up in a small town near the Minnesota/ North Dakota border. I lived there for most of my life. I moved to Nebraska and lived there for a year and then moved to Massachusetts. I have a Bachlors degree in Microbiology. 
 
+In my very little free time I like to read fantasy novels, crochet, and I love to bake. I also enjoy being outdoors but have not had much of a chance to explore any trails.
+
+Answer-two
 https://github.com/awalsh0517
 
 www.linkedin.com/in/andrea-walsh-b31694135
 
-In terms of technical skills they are very limited. I know basics of HTML and CSS.
\ No newline at end of file
+In terms of technical skills they are very limited. I know basics of HTML and CSS.
+master
```

Comment:
When resolving a conflict, git provides us with these markers so we know which branch something came from. We should remove them during conflict resolution since they are just git provided markers.